README.md

```diff
@@ -1,2 +1,34 @@
-## Dev-Mountain Solidity Contract
+# **Dev. Mountain Tech Festival(Thailand) 2022 - Solidity Contract**
 
+
+## What is Dev. Mountain Tech Festival  
+
+Dev. Mountain Tech Festival is Thailand tech events that setup at
+18-20 March 2022 Toscana Valley Khao Yai.This event is have many top-tier Thai developers as speaker and have many different topic for developers or anyone who interested in technology
```

Comment:
- This event is have ... -> This event has ... ตรงนี้ event เป็นเอกพจน์ (ไม่เติม s) คิดว่าใช้ has ไปเลย แล้วตอนนี้เราเล่า fact ปกติใช้ present simple ได้ เลยคิดว่าใช้แค่ has เป็น verb แท้ตัวเดียวพอแล้ว
- and have many different ... -> and has many different เพราะประธานคือ this event เหมือนกับประโยคแรก
- many different topic ... -> many different topics เพราะมีหลาย topic
ปล. ถ้าผมเข้าใจพลาดขออภัย